README.md

```diff
@@ -1,121 +1,115 @@
-[![PyPI version](https://badge.fury.io/py/tslearn.svg)](https://badge.fury.io/py/tslearn)
-[![Documentation Status](https://readthedocs.org/projects/tslearn/badge/?version=latest)](http://tslearn.readthedocs.io/en/latest/?badge=latest)
-[![Build Status](https://travis-ci.org/rtavenar/tslearn.svg?branch=master)](https://travis-ci.org/rtavenar/tslearn)
-[![Codecov](https://codecov.io/gh/rtavenar/tslearn/branch/master/graph/badge.svg)](https://codecov.io/gh/rtavenar/tslearn)
-[![Downloads](https://pepy.tech/badge/tslearn)](https://pepy.tech/project/tslearn)
-
-`tslearn` is a Python package that provides machine learning tools for the analysis of time series.
-This package builds on `scikit-learn`, `numpy` and `scipy` libraries.
-
-If you would like to contribute to `tslearn`, please have a look at [our contribution guidelines](CONTRIBUTING.md).
-
-# Dependencies
-
-```
-Cython
-numpy
-numba
-scipy
-scikit-learn
-joblib
-numba
-```
-
-If you plan to use the `shapelets` module, `keras` and `tensorflow` should also be installed.
-`h5py` is required for reading or writing models using the hdf5 file format.
+<!-- Our title -->
+<div align="center">
+  <h3>tslearn </h3>
+</div>
+
+<!-- Short description -->
+<p align="center"> 
+   "The machine learning toolkit for time series analysis in Python"
+</p>
+
+<!-- The badges -->
+<p align="center">
+    <a href="https://badge.fury.io/py/tslearn">
+        <img alt="PyPI" src="https://badge.fury.io/py/tslearn.svg">
+    </a>
+    <a href="http://tslearn.readthedocs.io/en/latest/?badge=latest">
+        <img alt="Documentation" src="https://readthedocs.org/projects/tslearn/badge/?version=latest">
+    </a>
+    <a href="https://travis-ci.org/rtavenar/tslearn">
+        <img alt="Build" src="https://travis-ci.org/rtavenar/tslearn.svg?branch=master">
+    </a>
+    <a href="https://codecov.io/gh/rtavenar/tslearn">
+        <img alt="Codecov" src="https://codecov.io/gh/rtavenar/tslearn/branch/master/graph/badge.svg">
+    </a>
+    <a href="https://pepy.tech/project/tslearn">
+        <img alt="Downloads" src="https://pepy.tech/badge/tslearn">
+    </a>
+</p>
+
+<!-- Draw horizontal rule -->
+<hr>
+
+<!-- Table of content -->
+
+| Section | Description |
+|-|-|
+| [Installing](#installation) | Installing the dependencies and tslearn |
+| [Getting started](#getting-started) | A quick introduction on how to use tslearn |
+| [Documentation](#documentation) | A link to our API reference and a gallery of examples |
```

Comment:
Optionally, we can leave out the Documentation section from the readme and just directly link to RTD

Reply:
I guess we can keep it like that so that the numbered sections fit well.